clients/admin-ui/src/features/privacy-requests/buttons/ReprocessButton.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Button,
+  ButtonProps,
+  forwardRef,
+  ListItem,
+  UnorderedList,
+} from "@fidesui/react";
+import { useState } from "react";
+
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+
+import { useAlert, useAPIHelper } from "../../common/hooks";
+import {
+  selectRetryRequests,
+  setRetryRequests,
+  useBulkRetryMutation,
+  useRetryMutation,
+} from "../privacy-requests.slice";
+import { PrivacyRequest } from "../types";
+
+type ReprocessButtonProps = {
+  buttonProps?: ButtonProps;
+  subjectRequest?: PrivacyRequest;
+};
+
+const ReprocessButton = forwardRef(
+  ({ buttonProps, subjectRequest }: ReprocessButtonProps, ref) => {
+    const dispatch = useAppDispatch();
+    const [isReprocessing, setIsReprocessing] = useState(false);
+    const { handleError } = useAPIHelper();
+    const { errorAlert, successAlert } = useAlert();
+
+    const { errorRequests } = useAppSelector(selectRetryRequests);
+    const [bulkRetry] = useBulkRetryMutation();
+    const [retry] = useRetryMutation();
+
+    const handleBulkReprocessClick = () => {
+      setIsReprocessing(true);
+      bulkRetry(errorRequests!)
```

Comment:
~It's not obvious to me that `errorRequests` must be defined, so `!` might not be safe. The function could do an undefined check and just return if it's blank for some reason.~

Oh, actually typechecking says this is always defined, so the exclamation point is redundant. (Though it could still introduce type unsafety if it became optional later.)